src/LibraryManager/Manifest.cs

```diff
@@ -164,17 +349,66 @@ public void AddVersion(string version)
                 return new ILibraryInstallationResult[] { LibraryInstallationResult.FromError(PredefinedErrors.VersionIsNotSupported(Version)) };
             }
 
+            IEnumerable<FileConflict> fileConflicts = null;
```

Comment:
Can you moce all this conflict detection logic + Creation of Errors outside of this method? This method should return ILibraryInstallationResult and we just check for this results status